### PR TITLE
Todo List

### PR DESCRIPTION
**Pull request by tiagoit**
Implemented a todo list using Vanilla JS:
- Modules
- Classes
- Static classes
- Router simulation (SPA Single Page Application)
- uuid NPM library
- webpack
- Bootstrap

### CODE REVIEW

**Review by bellom — COMMENTED** [review]
Reviewing !

**Review by bellom — COMMENTED** [review]
.

**Review by bellom — COMMENTED** [review]
.

File: src/app.service.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AppService {
+
```
Comment (bellom) [inline src/app.service.js]: kindly remove this function, we have no logic inside.

File: src/framework.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import AppService from './app.service';
+import DashboardController from './controllers/dashboard.controller';
+
+class Framework {
+  /**
```
Comment (bellom) [inline src/framework.js:4]: **Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face: **

**Review by bellom — CHANGES_REQUESTED** [review]
**Hi Team,**
**Great job :+1:  setting up the project :slightly_smiling_face: 

We still have some features to work on;
For some reason your app still lack some important features.
Kindly work on the following; 
#### Interface ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/ui_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list))


#### To-do item / Task ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/todo_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list))

When i open your app, am unable to add todo to the project created, and there's no `edit button` and `delete button` for todo. 
checkout the screenshot from your webpage.
image


**Kindly watch and listening carefully to the video and audio code review below;**

https://www.loom.com/share/4327d7fe06684c648d039b51765be326

**_Hope it helps clarify the issues we are having ......_**

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**


Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by tiagoit** [comment]
Hi @bellom, I guess eslint auto-fix may have broken the toy, working on it now.

**Review by BalintMendli — APPROVED** [review]
Hi @ferreirati @AndresFMoya ,

The app now works as expected. 👏 

It is a 👍 from me.

You can go ahead and merge the project.

**Pull request merged**